backend/src/appointment/database/models.py

```diff
@@ -179,7 +179,7 @@ class Appointment(Base):
     __tablename__ = 'appointments'
 
     id = Column(Integer, primary_key=True, index=True)
-    uuid = Column(UUIDType(native=False), default=uuid.uuid4(), index=True)
+    uuid = Column(UUIDType(native=False), default=uuid.uuid4, index=True, unique=True)
```

Comment:
SQLAlchemy also provides a Uuid type since 2.0, I think using that could prevent issues with default values maybe.
https://docs.sqlalchemy.org/en/20/core/type_basics.html#sqlalchemy.types.Uuid

Reply:
Also thanks for adding `unique=True`! How could we possibly miss that 😅

Reply:
We generally don't need uniqueness because most if not everything is scoped by the subscriber. But I felt like it'd be silly not to unique a uuid.

And I can make a ticket for that in the future!

Reply:
Ah right. Thank you!